components/LinkToInformationAboutRoadmap.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <span :class="$style.linkButton">
+    <a
+      :class="$style.externalLink"
+      href="https://www.bousai.metro.tokyo.lg.jp/1007942/index.html"
+      target="_blank"
+      rel="noopener noreferrer"
+    >
+      {{ $t('ロードマップについて') }}
+      <v-icon
+        :class="$style.externalLinkIcon"
+        size="1.5rem"
+        :aria-label="this.$t('別タブで開く')"
+        role="img"
+        :aria-hidden="false"
+      >
+        mdi-open-in-new
+      </v-icon>
+    </a>
+  </span>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+
+export default Vue.extend()
+</script>
+
+<style lang="scss" module>
+.linkButton {
+  display: inline-flex;
+  flex: 0 1 auto;
+
+  @include lessThan($small) {
+    margin-top: 4px;
+  }
+}
+
+.externalLink {
+  @include button-text('sm', 16);
+
+  color: $green-1 !important;
+  text-decoration: none;
```

Comment:
追加ですが、ここに `!important` がある影響で `variables.scss` の `<a>` 要素 `:hover` 時の `color: $white` 指定が効かなくなっているようなので、
```scss
  &:hover {
    color: $white !important;
  }
```
が必要かもしれません

```suggestion
  color: $green-1 !important;
  text-decoration: none;
  
  &:hover {
    color: $white !important;
  }
```

Reply:
なるほどです！vuetify対策をしたときに、確認わすれてました。